### DISCUSSION TITLE
Restart the match option

### DISCUSSION THREAD

**Discussion by Kiwi1444** [ikemen-engine/Ikemen-GO#2172] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

Once again, while pressing escape key and selecting the option to change character, I thought it would be nice to have the option to restart the match. Without having to go to select screen and select the same characters and the same stage.

**Comment by SuperFromND** [comment]
This is already possible via debug keys (F4 or Shift-F4), but it would be kinda nice to have it as a menu option in training mode for those playing on controller.

**Comment by potsmugen** [comment]
Could have both reset positions for Training mode and a rematch option in Versus modes.

**Reply by Kiwi1444** [reply]
And rematch option in watch mode, pls.

**Comment by wily-coyote** [comment, marked as answer]
Being able to restart the match from the menu seems to already be a feature in IKEMEN. It's just a matter of adding `menu.itemname.reload` or `menu.itemname.reset` (depending on your use-case) it to your system's `[Menu Info]` or `[Training Info]`.
https://github.com/ikemen-engine/Ikemen-GO/blob/aa9e6c270a3729e273256d34201caef38e488f0e/external/script/menu.lua#L207-L228
As for resetting position, there doesn't seem to be any existing feature. That could be particularly useful in Training or training-like gamemodes.
